src/System.IO/src/System.IO.builds

```diff
@@ -25,6 +25,12 @@
     <Project Include="redist\System.IO.depproj">
       <TargetGroup>netcore50aot</TargetGroup>
     </Project>
+    <Project Include="System.IO.csproj">
+      <TargetGroup>uap101aot</TargetGroup>
+    </Project>
+    <Project Include="System.IO.csproj">
```

Comment:
Is it necessary to introduce a uap101 non-aot build? Can we not use the other build for this?